components/json-ld.tsx

```
@@ -0,0 +1,55 @@
+      } : undefined,
+    })),
+  }
+
+  return (
+    <Script
+      id="json-ld"
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{
+        __html: JSON.stringify(jsonLd),
+      }}
+      strategy="afterInteractive"
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)